Mathlib/NumberTheory/ModularForms/LevelOne.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2024 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import Mathlib.NumberTheory.Modular
+import Mathlib.NumberTheory.ModularForms.SlashInvariantForms
+/-!
+# Level one modular forms
+
+This file contains results specific to modular forms of level one, ie. modular forms for `SL(2, ℤ)`.
+
+TODO: Add finite-dimensionality of these spaces of modular forms.
+
+-/
+
+open UpperHalfPlane ModularGroup SlashInvariantForm ModularForm Complex MatrixGroups
+
+lemma SlashInvariantForm.exists_norm_le {k : ℤ} (hk : k ≤ 0) {F : Type*} [FunLike F ℍ ℂ]
```

Comment:
Perhaps we should rename to `SlashInvariantForm.exists_one_half_le_im_and_norm_le`.